tests/integration-tests/test-cases/runTest-test.js

```diff
@@ -48,11 +48,8 @@ describe('Successfully run a custom test', function () {
         console.log('REPORT:', testReport);
     });
 
-    it('Test should finish successfully', function () {
+    it('Test should finish successfully', async function () {
         should.exist(testReport);
```

Comment:
maybe get here the aggregateReport and see that there were requests going out or something since `testReports = {}`

Reply:
it's integration so it won't really help
but it's tested in system tests

Reply:
added some nock validations